### DISCUSSION TITLE
I'm having a problem with the APK

### DISCUSSION THREAD

**Discussion by Jadecraft4** [FreeTubeApp/FreeTube#2909] [Q&A] [answered]
I don't know where to write this, but I didn't want to write a bug report if I can get it fixed in the discussions.
I've gotten the APK for Freetube from 
https://github.com/MarmadileManteater/FreeTubeCordova/releases
But when I import my subscriptions, it gives an error about how it can't get things from the feed or something. 
And it doesn't seem like it's just importing my subscriptions, I think what breaks it is too many subscriptions, which might overload something? I tried putting my subscriptions manually, but this also broke it. What's broken about it is when I close the app, and open it again, it shows a blank page of white. 

Any help is appreciated.

**Comment by senolfndk** [comment]
i get the same error
![Screenshot_20221205-212104_FreeTube](https://user-images.githubusercontent.com/52818836/205713044-3445a7cd-21fe-4154-87d2-2a2859fabe74.jpg)

**Comment by MarmadileManteater** [comment, marked as answer]
This is not a known issue, so I would suggest [making a 🪲 bug report under the FreeTubeCordova repo](https://github.com/MarmadileManteater/FreeTubeCordova/issues/new?assignees=&labels=bug&template=bug_report.yaml&title=%5BBug%5D%3A+) in order to help document this problem.

My absolute first 🤔guess as to what might be causing this would be `localStorage` size limitations. In FreeTubeCordova, subscriptions are stored using `localStorage` instead of a local db file, so potentially, if you had a big enough subscriptions list, this could cause the data stored in `localStorage` to be shortened in a way which could cause the app to fail to load due to a deserialization error. However, this is just an educated guess without replicating the issue myself first.
